qiskit/circuit/classical/expr/__init__.py

```diff
@@ -86,10 +86,18 @@
 The functions and methods described in this section are a more user-friendly way to build the
 expression tree, while staying close to the internal representation.  All these functions will
 automatically lift valid Python scalar values into corresponding :class:`Var` or :class:`Value`
-objects, and will resolve any required implicit casts on your behalf.
+objects, and will resolve any required implicit casts on your behalf.  If you want to directly use
+some scalar value as an :class:`Expr` node, you can manually lift it yourself.
```

Comment:
Maybe use a `ref` to `lift()` here. The verb might not have sufficient meaning for people unless they know about the function or are fairly well versed in lambda calculus or other programming language concepts. Although I guess the lift autofunction is right below this, but I just know that this term will be unfamiliar to most people reading the docs.